composer.json

```
@@ -16,11 +16,11 @@
     "ext-json": "*",
     "guzzlehttp/guzzle": "~6.3",
     "beberlei/assert": "~2.7|~3.0",
-    "flix-tech/avro-php": "^3.0"
+    "flix-tech/avro-php": "^4.0"
```

```suggestion
    "flix-tech/avro-php": "^3.0|^4.0"
```
@tPl0ch since the bug shouldn't affect the schema registry client, let me know if i should change it to support both versions